libcraft/items/src/item_stack.rs

```diff
@@ -299,3 +342,105 @@ impl Display for ItemStackError {
 }
 
 impl Error for ItemStackError {}
+
+impl ItemStackMeta {
+    pub fn new(item: Item) -> Self {
+        Self {
+            title: item.name().to_owned(),
+            lore: "".to_owned(),
+            damage: None,
+            repair_cost: None,
+            enchantments: vec![],
+        }
+    }
+}
+
+pub struct ItemStackBuilder {
+    item: Item,
+    count: NonZeroU32,
```

Comment:
Why is this a U32 when count is a `u8`?

Reply:
count is a u32.

Reply:
I think Minecraft uses i32?

Reply:
It might be the case that ItemStacks should allow for negative stack sizes. However, 
i believe this is outside the scope of this refactor.  That was how ItemStack were modelled when we started. This "issue" should be revisited on a separate pull request.